desktop/api/src/main/java/org/datacleaner/util/IconUtils.java

```diff
@@ -188,6 +188,10 @@
     public static final String DICTIONARY_TEXTFILE_IMAGEPATH = "images/model/dictionary_textfile.png";
     public static final String DICTIONARY_DATASTORE_IMAGEPATH = "images/model/dictionary_datastore.png";
 
+    public static final String NEWS_CHANNEL = "images/news/news_channel_16.png";
+    public static final String NEWS_CHANNEL_ORANGE = "images/news/news_channel_28.png";
```

Comment:
Maybe not the best name: It's more important what it is used for (or better, what it represents), than what it looks like. Then we can change the image later without the constant name not making sense anymore, or having to change names.